ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -24,27 +24,29 @@ function FeatureCard (props: FeatureUIProps): JSX.Element {
   const classes = useStyles()
 
   return (
-    <Link className={classes.cardLink} to={props.route} >
-      <Card className={`${classes.root}`} variant="outlined" tabIndex={props.data.ordinality}>
-        <CardContent>
-          <Grid container>
-            <Grid item xs={12}>
-              <div className={classes.centered}>
-                {props.icon}
-              </div>
-              <div className={classes.centered}>
-                <Typography className={classes.title} color="textPrimary" gutterBottom>
-                  {props.data.title}
-                </Typography>
-                <Typography variant="body2" component="p" color="textSecondary">
-                  {props.data.description}
-                </Typography>
-              </div>
+    <RouterLink className={classes.cardLink} to={props.route} tabIndex={-1}>
+      <Card variant='outlined'>
+        <CardActionArea>
+          <CardContent className={`${classes.cardContent}`}>
```

Comment:
I moved the color and height CSS settings to `CardContent` to ensure that the parent `CardActionArea` covers all the content.